delivery/models.py

```diff
@@ -0,0 +1,45 @@
+from flask import jsonify, request
+from flask.views import MethodView
+from dataclasses import dataclass
+from marshmallow import ValidationError
+from delivery.repositories import AbstractRestaurantRepository, MemoryRestaurantRepository
+
+
+@dataclass
+class Restaurant:
+    name: str
+    address: str
+    work_time: str
+    phone_number: str
+    id: int = 0
+
+
+class RestaurantEndpoint(MethodView):
```

Comment:
Flask routes should be defined in another file, `model.py` is just to define the data that the application is working with

Reply:
fixed